docs/rkt/README.md

```diff
@@ -0,0 +1,34 @@
+# Integrating the Calico Network Plugin with rkt
+
+This guide will describe the configuration required to use the Calico network plugin in your rkt deployment.
+
+## Using rkt Plugins
+
+The CoreOS [documentation](https://github.com/coreos/rkt/blob/master/Documentation/networking.md) for Network Plugins will walk you through the basics of setting up networking in rkt.
+
+## Requirements
+
+* A working [etcd](https://github.com/coreos/etcd) service
+* A build of `calicoctl` after [projectcalico/calico-docker@10460cc405](https://github.com/projectcalico/calico-docker/commit/10460cc405f5aa4bc9ccb1fcaf8760088ae1ebf9)
+* Though Calico is capable of networking rkt containers, our core software is distributed and deployed in a [docker container](https://github.com/projectcalico/calico-docker/blob/master/docs/getting-started/default-networking/Demonstration.md). While we work on native rkt support, you will need to run Calico in Docker before starting your rkt containers. This can be easily done wtih `calicoctl` by running the following command: `sudo calicoctl node --ip=<IP> --rkt`
+
+## Installing
+
+* Running `calicoctl node` with the `--rkt` flag will start the calico/node process and automatically install the plugin for you. Alternatively you can download the [plugin binary](https://github.com/projectcalico/calico-rkt/releases/) yourself and move it to the rkt plugin directory.
```

Comment:
Let's give them a copy-and-paste-able command to run.. e.g:

We recommend installing the plugin using the following `calicoctl` command.  This will run our Docker agent, as well as install the plugin.
```
calicoctl node --ip=<IP> --rkt
```

Including the `--rkt` command downloads the `rkt` plugin and moves it to the correct location.  You can manually install the plugin by downloading one of our releases in the [calico-cni](https://github.com/projectcalico/calico-rkt/releases/) repo